github/code-scanning.go

```diff
@@ -34,13 +67,15 @@ type Alert struct {
 	RuleID          *string    `json:"rule_id,omitempty"`
 	RuleSeverity    *string    `json:"rule_severity,omitempty"`
 	RuleDescription *string    `json:"rule_description,omitempty"`
+	Rule		        *Rule	     `json:"rule,omitempty"`
 	Tool            *Tool      `json:"tool,omitempty"`
 	CreatedAt       *Timestamp `json:"created_at,omitempty"`
 	Open            *bool      `json:"open,omitempty"`
 	ClosedBy        *User      `json:"closed_by,omitempty"`
 	ClosedAt        *Timestamp `json:"closed_at,omitempty"`
 	URL             *string    `json:"url,omitempty"`
 	HTMLURL         *string    `json:"html_url,omitempty"`
+	MostRecentInstance *MostRecentInstance `json:"most_recent_instance,omitempty"`
```

Comment:
Do you also want to add the other missing `Alert` fields here as well?

* `"dismissed_by"`
* `"dismissed_at"`
* `"dismissed_reason"`
* `"instances_url"`

?

Reply:
Ofcourse, I added these 4 new fields for the `Alert` struct as well :)
```
"dismissed_by"
"dismissed_at"
"dismissed_reason"
"instances_url"```